blankiety_galantow/core/game_master.py

```diff
@@ -88,20 +99,23 @@ async def handle_cards_reveal(self, data):
             await player.send_json(message)
 
     async def timer_start(self):
-        if self.selecting_time != self.settings.selecting_time:
-            self.selecting_time = self.settings.selecting_time
-        self.timer_start_time = time.time()
-        if self.timer is not None:
-            self.timer.cancel()
-            try:
-                await self.timer.task
-            except asyncio.CancelledError:
-                pass
-            except KickException:
-                raise
-        self.timer = Timer(self.selecting_time, self.handle_timeout)
-        
+        if not self.paused:
+            if self.selecting_time != self.settings.selecting_time:
+                self.selecting_time = self.settings.selecting_time
+            if self.timer_start_time == 0:
+                self.timer_start_time = time.time()
+            if self.timer is not None:
+                self.timer_start_time = time.time()
+                self.timer.cancel()
+                try:
+                    await self.timer.task
+                except asyncio.CancelledError:
+                    pass
+                except KickException:
```

Comment:
Łapmy wszystko i rzucajmy ponownie.
```suggestion
                except Exception:
```